content/changes.md

```diff
@@ -8,6 +8,36 @@ summary: 'Learn about new features, fixes, and other improvements'
 
 Learn about notable new features, fixes, and other improvements to the application!
 
+## February 29, 2024  
+
+### Improvements 
+
+- **Dart 3.3** 
+
+  Dart 3.3 is now supported in Qualified.
+
+- **Qualified docs repository is public** 
```

Comment:
Looks good, although this item about the docs being public is still in. I assume you want to keep it--maybe ask Jake if you have concerns about removing it. Although we're inviting people to contribute, it's sort of a formality--I don't anticipate anyone will do so and the intent behind going public was basically just to save money on reviewer seats.